src/main/java/frc/trigon/robot/subsystems/climber/simulationclimber/SimulationClimberConstants.java

```diff
@@ -0,0 +1,77 @@
+package frc.trigon.robot.subsystems.climber.simulationclimber;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import frc.trigon.robot.simulation.ElevatorSimulation;
+import frc.trigon.robot.simulation.MotorSimulationConfiguration;
+import frc.trigon.robot.subsystems.climber.ClimberConstants;
+
+public class SimulationClimberConstants {
+    private static final int MOTOR_AMOUNT = 1;
+    private static final double
+            RIGHT_MOTOR_GEAR_RATIO = 10,
+            LEFT_MOTOR_GEAR_RATIO = 10;
```

Comment:
use the same ratio from the real io implementation. Move the gear ratio to the CilmberConstants